### ISSUE TITLE
Crop not working on PhoneGap Android Application

### ISSUE THREAD

**Issue by abhisheknjain** [alexk111/ngImgCrop#80]
Hi All,
I am working on a PhoneGap application. This library works fine on a desktop web browser but doesn't work when I try to convert into a PhoneGap mobile application. 

Has anyone here faced the same issue. Can anybody help me here. I am in real trouble.

Thanks in advance. :)

**Comment by shaoner** [comment]
Nop, works fine with my cordova app

**Comment by rochapablo** [comment]
It's not working if I access the website in my phone (android, wp, iphone).

**Comment by ppss198** [comment]
it works fine on a desktop web browser, android browser, iphone browser .but  doesn't work when i  convert into a apk application.

**Comment by ketansp** [comment]
Yup. It works for browser, android browser. Will let you know about app in a couple of days.

**Comment by rochapablo** [comment]
I'm sorry, I was using an old version.
My bad.

**Comment by darmie** [comment]
Hi, did anybody get it to work in a compiled cordova app?

**Comment by ppss198** [comment]
it work in a compiled cordova app , mobile-config.js need
add App.accessRule("blob:*");  it only dont work in android 4.5 web browser
. and today is chinese new year .happy new year

**Comment by darmie** [comment]
Thanks @alexk111 , I got it to work already.

**Comment by pramodwerea** [comment]
hi still I found this problem ? in phonegap I used 

`    <allow-navigation href="*" />
    <allow-navigation href="blob:*" />
    <allow-navigation href="data:*" />
    <allow-intent href="*" />`

but still it is not working.
